setup.py

```diff
@@ -20,7 +20,7 @@
     author_email='nmarais@ska.ac.za',
     license='GPL',
     packages=find_packages(exclude=['ez_setup']),
-    install_requires=['Nose>=0.11.0', 'traceback2', 'ansicolors'],
+    install_requires=['Nose>=0.11.0', 'traceback2', 'ansicolors', 'numpy'],
```

Comment:
I see it is used in `plugin.py`, so we do need it.  In our other [CAM deployment requirements](https://github.com/ska-sa/katdeploy/blob/master/trunk-requirements.txt), we use `numpy>=1.6.0`, but that restriction probably doesn't matter since that they are at 1.13 already.